libs/application/templates/official-journal-of-iceland/src/screens/InvolvedPartyScreen.tsx

```
@@ -0,0 +1,107 @@
+      <Box width="half">
+        <OJOISelectController
+          disabled={disableSelect}
+          loading={loading}
+          name={InputFields.advert.involvedPartyId}
+          label={f(involvedParty.general.section)}
+          options={options}
+          applicationId={application.id}
+          defaultValue={defaultValue}
+          placeholder={involvedParty.inputs.select.placeholder}
+          onChange={() =>
+            setSubmitButtonDisabled && setSubmitButtonDisabled(false)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)